packages/overlay/src/integrations/sentry/components/EventBreadcrumbs.tsx

```diff
@@ -21,9 +23,9 @@ function extractBreadcrumbs(event: SentryEvent): Breadcrumb[] | undefined {
 
 export default function EventBreadcrumbs({ event }: { event: SentryEvent }) {
   const breadcrumbs = extractBreadcrumbs(event);
-  if (!breadcrumbs) {
+  if (!breadcrumbs || (Array.isArray(breadcrumbs) && breadcrumbs.length === 0)) {
```

Comment:
I think we don't need the isArray check as `breadcrumbs` can either be undefined or an array:

```suggestion
  if (!breadcrumbs || !breadcrumbs.length)) {
```

Reply:
Scratch that, I think the type signature of `extractBreadcrumbs` is outdated. It already only returns `Breadcrumb[]` so let's change the signature and simplify this check.

Reply:
@Lms24 I think the suggestion you made is needed.
it was a miss from my end, extractbreadcrumbs always return array, so I think there's no need to check whether breadcrumbs is Arrray.

thanks